internal/provisioner/cluster.go

```diff
@@ -98,13 +104,24 @@ func CreateCluster(provider, s3StateStore, size string, zones []string, logger l
 		return err
 	}
 
+	if waitForReady {
+		logger.WithField("dns", dns).Infof("waiting up to %d seconds for k8s cluster to become ready...", maxReadynessWaitSeconds)
```

Comment:
Do we need `WithField("dns", dns)` here? If we do, let's just modify `logger` directly on line 63. But I'd argue we don't -- the cluster id embedded is already sufficient. The DNS added on line 63 was more of a "hey, this is the DNS entry!".

Reply:
Yeah, that's probably overly verbose. I will change it.